tests/test_reward_management.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import brownie
+from brownie import (
+    xQI,
+    Qi,
+    interface,
+    accounts,
+    Wei,
+    Contract,
+    chain,
+    BaseRewardPool,
+    MainStaking,
+)
+
+
+def test_xqi_required(setup_contracts, fn_isolation):
```

Comment:
Thius is in fact a test that looks like the real use case, there is no revert so the test may no be properly named